pyfar/signals/files.py

```diff
@@ -389,33 +392,43 @@ def head_related_impulse_responses(
     tolerance_rad = 0.1 / 180 * np.pi
 
     # get indices of source positions
-    if position == "horizontal":
+    if isinstance(position, str) and position == "horizontal":
         idx = sources.elevation == 0
-    elif position == "median":
+    elif isinstance(position, str) and position == "median":
         idx = sources.lateral == 0
         idx = np.where(idx)[0]
         # sort positions according to polar angle
         polar = sources.polar[idx].flatten()
         idx = (idx[np.argsort(polar)], )
     else:
-        idx = []
-        for pos in position:
-            find = pf.Coordinates.from_spherical_elevation(
-                pos[0] / 180 * np.pi, pos[1] / 180 * np.pi, 1.7)
-            idx_current, distance = sources.find_nearest(
-                find, distance_measure='spherical_radians')
-            if distance < tolerance_rad:
-                idx.append(idx_current[0])
-            else:
-                raise ValueError((
-                    f"HRIR for azimuth={pos[0]} and elevation={pos[1]} degrees"
-                    " is not available. See documentation for more "
-                    "information."))
+
+        if isinstance(position, pf.Coordinates):
+            # force radius to be equal to those of HRIRs
+            position.radius = 1.7
+        else:
+            # convert to pyfar coordinates for convenience
+            position = np.atleast_2d(position) / 180 * np.pi
+            position = pf.Coordinates.from_spherical_elevation(
+                position[:, 0], position[:, 1], 1.7)
```

Comment:
```suggestion
                position[..., 0], position[..., 1], 1.7)
```
would be more save in case of multidimensional arrays